app/assets/javascripts/discourse/controllers/composer.js.es6

```diff
@@ -909,19 +910,24 @@ export default Controller.extend({
       isWarning: false
     });
 
-    if (opts.usernames && !this.get("model.targetUsernames")) {
-      this.set("model.targetUsernames", opts.usernames);
+    if (!this.model.targetRecipients) {
+      if (opts.usernames) {
+        deprecated("`usernames` is deprecated, use `recipients` instead.");
+        this.model.set("targetRecipients", opts.usernames);
```

Comment:
I think all those `.set()` should now be replaced with the more direct `this.model.targetRecipients = opts.usernames`.